app/src/main/java/org/fossasia/susi/ai/chat/STTfragment.kt

```diff
@@ -122,6 +122,9 @@ class STTfragment : Fragment() {
 
     override fun onPause() {
         super.onPause()
+        if ( activity is ChatActivity) {
+            (activity as ChatActivity).enableVoiceInput()
+        }
         (activity as ChatActivity).fabsetting.show()
```

Comment:
Move this inside the if block as well and remove the cast.

Reply:
move the fabsetting.show()? shouldn't I do it in another PR? wouldn't it confuse others if they look at it later?

Reply:
also @iamareebjamal I think the cast is required by the compiler, it's an error if we remove the special cast.

Reply:
I thought it should work through Kotlin smart cast. Let me check and get back to you